packages/app-project/src/screens/ProjectAboutPage/components/TeamMember.js

```diff
@@ -80,9 +80,9 @@ const TeamMember = ({ user, router }) => {
         />
         {user.roles &&
           user.roles.length &&
-          user.roles.map(role => (
+          user.roles.map((role, index) => (
             <StyledRole
-              key={role}
+              key={`${role}${index}`}
```

Comment:
Hm, I thought the role resource would have a unique id, perhaps we could use here? I'm not sure exactly if that's being preserved from the original API response by the time it gets here though?

Reply:
The user object for each team member does have an id, but at this point `role` is a string. You're right though, it should be unique and index is not needed.